### PR TITLE
Adds support for creating/updating entries

### PR DESCRIPTION
**Pull request by monicatie**
Fixes https://github.com/contentful-labs/contentful-go/issues/15

Borrows code from https://github.com/contentful-labs/contentful-go/pull/25 
Didn't see any reason why that was closed...so trying again!

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/contentful-labs/contentful-go
+module github.com/outer-labs/contentful-go
```
Comment (gunturaf) [inline go.mod:1]: I believe this line should not be merged with upstream repo

File: entry.go [hunk 2]
```diff
@@ -90,6 +92,46 @@ func (service *EntriesService) Get(spaceID, entryID string) (*Entry, error) {
 	return &entry, err
 }
 
+
+// Upsert updates or creates a new space
```
Comment (gunturaf) [inline entry.go:96]: The comment should be `// Upsert updates or creates a new Entry` no?

Comment (monicatie) [reply entry.go:96]: ah yes, thanks for the catch!

**Comment by gunturaf** [comment]
@Khaledgarbaya please consider reviewing this feature as currently there is no convenient way to create an Entry using this SDK. CMIIW?

**Pull request merged**